### DISCUSSION TITLE
Considering Turing College courses, feedback needed

### DISCUSSION THREAD

**Discussion by AverageReactEnjoyer** [community/community#38180] [Other Feature Feedback, Questions, & Ideas]
Hello!

I've recently heard about this Bootcamp'ish coding course platform - Turing College. I love the structure, and how they offer learning plans and a 1-on-1 mentorship. Has anyone here tried the platform, and could provide any more in-depth information? Would highly appreciate it!

Link for reference: https://www.turingcollege.com/

**Comment by david-engelmann** [comment]
Checking out their socials, they seem pretty small. Make sure you aren't getting into any predatory contracts
